sklearn/metrics/_classification.py

```diff
@@ -670,7 +698,17 @@ class labels [2]_.
     .. [3] `Wikipedia entry for the Cohen's kappa
             <https://en.wikipedia.org/wiki/Cohen%27s_kappa>`_.
     """
-    confusion = confusion_matrix(y1, y2, labels=labels, sample_weight=sample_weight)
+    y_type, y1, y2 = _check_targets(y1, y2)
+    if y_type == "binary":
+        # we can set `pos_label` to any class labels because the computation of MCC
+        # is symmetric and invariant to `pos_label` switch.
+        pos_label = y1[0]
+    else:
+        pos_label = None
+
+    confusion = confusion_matrix(
+        y1, y2, labels=labels, pos_label=pos_label, sample_weight=sample_weight
+    )
```

Comment:
not sure why this part is needed if `confusion_matrix` does deduction of the `pos_label` correctly. Shouldn't the score take the `pos_label` instead?

Reply:
If the labels are strings, then there is an ambiguity in the confusion matrix regarding what is the positive label. Thus, it would raise an error. However, the Cohen Kappa score is symmetric so you can pass whatever positive label and you should not let the error to be raised.